tests/core/driver/test_endpoint_driver.py

```diff
@@ -4,7 +4,7 @@
 import pytest
 
 from lahja import BaseEvent, ConnectionConfig
-from lahja.tools.driver import (
+from lahja.tools.drivers import (
```

Comment:
There's no `lahja.tools.driver` package/module. I've no idea how CI didn't complain about this before?

Reply:
It looks like we forgot to actually run this test

https://github.com/ethereum/lahja/blob/b34700f3bdb040b91a943544f914109c2b0b883d/tox.ini#L27-L29

Reply:
It looks like we don't run the tests/core/common tests either. I've changed tox.ini in an attempt to correct that but they're still not being executed on CI. Any idea what's wrong?

Reply:
Good catch! 

>I've changed tox.ini in an attempt to correct that but they're still not being executed on CI. Any idea what's wrong?

They also have to be configured in `circleci/config.yml`

https://github.com/ethereum/lahja/blob/b34700f3bdb040b91a943544f914109c2b0b883d/.circleci/config.yml#L78-L89